test/undo.ts

```diff
@@ -14,11 +14,15 @@ import { sleep } from './helpers/sleep';
 useSpectron();
 
 async function undo(t: TExecutionContext) {
-  await t.context.app.client.click('.fa-undo');
+  t.context.app.client.keys(['Control', 'z']);
+  t.context.app.client.keys('Control');
+  await sleep(500);
```

Comment:
add await for sending keys instead of sleep
```
await t.context.app.client.keys(['Control', 'z']);
```

Reply:
keys doesn't return a promise it cant be awaited

Reply:
Just looked into the source code. It returns promise but seems like d.ts files are incorrect.
Try to use a typecasting here
```
await ((this.client.keys(['Control', 'z']) as any) as Promise<any>); 
```